vips/vip-256/bscmainnet.ts

```diff
@@ -0,0 +1,343 @@
+    amount: parseUnits("245187.19242229", 8).toString(),
+    address: "0x86aC3974e2BD0d60825230fa6F355fF11409df5c",
+  },
+  {
+    symbol: "vUNI",
+    amount: parseUnits("2497.74135186", 8).toString(),
+    address: "0x27FF564707786720C71A2e5c1490A63266683612",
+  },
+  {
+    symbol: "vFDUSD",
+    amount: parseUnits("10355.09068368", 8).toString(),
+    address: "0xC4eF4229FEc74Ccfe17B2bdeF7715fAC740BA0ba",
+  },
+  {
+    symbol: "vUSDC",
+    amount: parseUnits("1251038.96782643", 8).toString(),
+    address: "0xecA88125a5ADbe82614ffC12D0DB554E2e2867C8",
+  },
+  {
+    symbol: "vDOT",
+    amount: parseUnits("55089.83198008", 8).toString(),
+    address: "0x1610bc33319e9398de5f57B33a5b184c806aD217",
+  },
+  {
+    symbol: "vADA",
+    amount: parseUnits("662029.93490011", 8).toString(),
+    address: "0x9A0AF7FDb2065Ce470D72664DE73cAE409dA28Ec",
+  },
+  {
+    symbol: "vXRP",
+    amount: parseUnits("534654.72490775", 8).toString(),
+    address: "0xB248a295732e0225acd3337607cc01068e3b9c10",
+  },
+  // {
+  //     symbol: "vTUSDOLD",
+  //     amount: parseUnits("234046.29380374", 8).toString(),
+  //     address: "0x08CEB3F4a7ed3500cA0982bcd0FC7816688084c3"
+  // },
+  {
+    symbol: "vMATIC",
+    amount: parseUnits("255028.0497963", 8).toString(),
+    address: "0x5c9476FcD6a4F9a3654139721c949c2233bBbBc8",
+  },
+  {
+    symbol: "vLINK",
+    amount: parseUnits("13919.2457489", 8).toString(),
+    address: "0x650b940a1033B8A1b1873f78730FcFC73ec11f1f",
+  },
+  {
+    symbol: "vBETH",
+    amount: parseUnits("70.15886616", 8).toString(),
+    address: "0x972207A639CC1B374B893cc33Fa251b55CEB7c07",
+  },
+  {
+    symbol: "vDOGE",
+    amount: parseUnits("1940822.36351268", 8).toString(),
+    address: "0xec3422Ef92B2fb59e84c8B02Ba73F1fE84Ed8D71",
+  },
+  {
+    symbol: "vFIL",
+    amount: parseUnits("25756.68067599", 8).toString(),
+    address: "0xf91d58b5aE142DAcC749f58A49FCBac340Cb0343",
+  },
+  {
+    symbol: "vTUSD",
+    amount: parseUnits("2576.73315234", 8).toString(),
+    address: "0xBf762cd5991cA1DCdDaC9ae5C638F5B5Dc3Bee6E",
+  },
+  {
+    symbol: "vWBETH",
+    amount: parseUnits("0.89365885", 8).toString(),
+    address: "0x6CFdEc747f37DAf3b87a35a1D9c8AD3063A1A8A0",
+  },
+  {
+    symbol: "vDAI",
+    amount: parseUnits("27071.86465849", 8).toString(),
+    address: "0x334b3eCB4DCa3593BCCC3c7EBD1A1C1d1780FBF1",
+  },
+  {
+    symbol: "vBUSD",
+    amount: parseUnits("10908.57803433", 8).toString(),
+    address: "0x95c78222B3D6e262426483D42CfA53685A67Ab9D",
+  },
+  {
+    symbol: "vBCH",
+    amount: parseUnits("21.1512832", 8).toString(),
+    address: "0x5F0388EBc2B94FA8E123F404b79cCF5f40b29176",
+  },
+  {
+    symbol: "vSXP",
+    amount: parseUnits("925.79433296", 8).toString(),
+    address: "0x2fF3d0F6990a40261c66E1ff2017aCBc282EB6d0",
+  },
+];
+
+export const BEP20Transfers = [
+  {
+    symbol: "BTCB",
+    amount: BigNumber.from(parseUnits("3.5029", 18)).add(parseUnits("1.06876", 18)).toString(),
```

Comment:
parseUnits should return BigNumber anyway
```suggestion
    amount: parseUnits("3.5029", 18).add(parseUnits("1.06876", 18)).toString(),
```

Reply:
https://github.com/VenusProtocol/vips/pull/200/commits/8a2d7984d74e0aadb8ddcec54eb3fe6a5a47cd7c